src/openrct2/ride/coaster/FlyingRollerCoaster.cpp

```diff
@@ -14594,22 +14617,22 @@ static void flying_rc_track_left_flyer_twist_up(
                 case 0:
                     PaintAddImageAsParentRotated(
                         session, direction, session.TrackColours[SCHEME_TRACK] | 27430, { 0, 6, height - 5 }, { 32, 20, 3 },
-                        { 0, 6, height + 8 });
+                        { 0, 6, height + 6 });
```

Comment:
This modifies existing code, which needs very good justification, could you elaborate?

Reply:
I have made a mistake in a previous PR which makes the vehicle clip through the rails. This change will make sure that the bug that was fixed originally doesn't come back, but also that the vehicle no longer clips.